From b62ef58d6cf51680e19e2ab65997e13588518fc0 Mon Sep 17 00:00:00 2001
From: Alfred Chen <cchalpha@gmail.com>
Date: Thu, 7 Oct 2021 20:15:28 +0000
Subject: [PATCH 216/273] sched/alt: Fix UP compilation issue.

Refine the code in rq_load_update() and fix UP compilation issue.
---
 kernel/sched/alt_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/sched/alt_core.c b/kernel/sched/alt_core.c
index 56aed2b1e42c..80faf3000851 100644
--- a/kernel/sched/alt_core.c
+++ b/kernel/sched/alt_core.c
@@ -603,7 +603,7 @@ static inline void rq_load_update(struct rq *rq)
 	u64 delta = min(LOAD_BLOCK(time) - LOAD_BLOCK(rq->load_stamp),
 			RQ_LOAD_HISTORY_BITS - 1);
 	u64 prev = !!(rq->load_history & CURRENT_LOAD_BIT);
-	u64 curr = !!cpu_rq(rq->cpu)->nr_running;
+	u64 curr = !!rq->nr_running;
 
 	if (delta) {
 		rq->load_history = rq->load_history >> delta;
-- 
2.36.1.74.g277cf0bc36

